src/playwright-runner.js

```diff
@@ -127,76 +127,61 @@ async function run (nodeBin, runCfgPath, suiteName) {
   const runCfg = await loadRunConfig(runCfgPath);
   runCfg.path = runCfgPath;
   const cwd = process.cwd();
-  let defaultArgs = {
-    param: {
-      headful: process.env.SAUCE_VM ? true : false,
-      video: shouldRecordVideo(),
-    },
-    reporter: 'junit,line',
-  };
-  let args = _.defaultsDeep(defaultArgs, {
-    output: path.join(cwd, '__assets__'),
-  });
 
   const suite = _.find(runCfg.suites, ({name}) => name === suiteName);
   if (!suite) {
     throw new Error(`Could not find suite named '${suiteName}'`);
   }
-  let env = {
-    ...process.env,
-    ...suite.env,
-    FOLIO_JUNIT_OUTPUT_NAME: path.join(cwd, '__assets__', 'junit.xml')
-  };
-  args = _.defaultsDeep(suite, args);
 
-  const folioBin = path.join(__dirname, '..', 'node_modules', 'folio', 'cli');
-  const procArgs = [folioBin];
-  const excludeParams = ['name', 'platform-name', 'browser-name', 'playwright-version', 'env'];
+  const projectPath = path.dirname(runCfg.path);
+  if (!fs.existsSync(projectPath)) {
+    throw new Error(`Could not find projectPath directory: '${projectPath}'`);
+  }
+
+  // Copy playwright configuration to project folder.
+  // As we curently don't support playwright configuration yet,
+  //   and set a configuration file automatically change the rootDir.
+  //   this is a workaround.
```

Comment:
Improved the wording a little. Also removed the `rootDir` sentence, since I didn't understand it or saw any references made in the code to that 🤔 
```suggestion
  // Copy our own playwright configuration to the project folder,
  // as we currently don't support having a user provided playwright configuration yet.
```

Reply:
@alexplischke  Let me try to give a more clear explanation of what I was trying to explain concerning `rootDir`.

Here is a local way to simulate behaviour from the runner on VMs.
project & runner are not the same directory.

Initially I added the `playwright.config.js` in the runner root directory and specified it using `--config`.
By doing that, `playwright test` command discards the current working directory and assume that the `rootDir` / `projectDir` is where is located the `playwright.config.js`.

Example:
```
{17:21}~/Projects/playwright-troubleshooting/project ➭ export NODE_PATH=`realpath ../runner/node_modules`
{17:21}~/Projects/playwright-troubleshooting/project ➭ ls -1 ../
project
runner
{17:21}~/Projects/playwright-troubleshooting/project ➭ ../runner/node_modules/.bin/playwright test

Running 1 test using 1 worker

  ✓ tests/test.spec.js:18:1 › [chromium] Test within PROJECT folder (7s)


  1 passed (8s)
{17:22}~/Projects/playwright-troubleshooting/project ➭ ../runner/node_modules/.bin/playwright test --config ../runner/playwright.config.js
Using config at /Users/xxx/Projects/playwright-troubleshooting/runner/playwright.config.js

Running 1 test using 1 worker

  ✓ tests/test.spec.js:18:1 › [chromium] Test within RUNNER folder (6s)


  1 passed (7s)
{17:22}~/Projects/playwright-troubleshooting/project ➭
```

As there is no option to enable video record outside of the `playwright.config.js` file, there is no other solution as creating that file in the project folder.

(And this part is not documented in playwright docs).

Is that a better explanation ?

Reply:
Ah interesting, thanks! 😄 Yeah, more clear now 👍 
It's just not very clear from the inline docs. Based on my understanding now, I'd reword it like this:
```
  // Copy our own playwright configuration to the project folder,
  // as we currently don't support having a user provided playwright configuration yet. Setting a config file
  // also automatically changes the root directory for playwright to the directory that contains the config file.
```